superset-frontend/plugins/plugin-chart-echarts/src/Gauge/transformProps.ts

```diff
@@ -100,10 +100,11 @@ export default function transformProps(
     filterState,
     theme,
     emitCrossFilters,
+    datasource,
   } = chartProps;
 
   const gaugeSeriesOptions = defaultGaugeSeriesOption(theme);
-
+  const { verboseMap } = datasource;
```

Comment:
This is needed for the proposed code change below:
```suggestion
  const { verboseMap = {} } = datasource;
```